src/contracts/gen.rs

```diff
@@ -0,0 +1,47 @@
+use crate::types::user_operation::UserOperation as UserOp;
+use ethers::contract::abigen;
+abigen!(
+    AggregatedAccount,
+    "$OUT_DIR/IAggregatedAccount.sol/IAggregatedAccount.json"
+);
+
+abigen!(Aggregator, "$OUT_DIR/IAggregator.sol/IAggregator.json");
+
+abigen!(
+    Create2Deployer,
+    "$OUT_DIR/ICreate2Deployer.sol/ICreate2Deployer.json"
+);
+
+abigen!(EntryPointAPI, "$OUT_DIR/IEntryPoint.sol/IEntryPoint.json");
+
+abigen!(Paymaster, "$OUT_DIR/IPaymaster.sol/IPaymaster.json");
+
+abigen!(
+    StakeManager,
+    "$OUT_DIR/IStakeManager.sol/IStakeManager.json"
+);
+
+abigen!(Account, "$OUT_DIR/IAccount.sol/IAccount.json");
+
+abigen!(
+    UserOperation,
+    "$OUT_DIR/UserOperation.sol/UserOperationLib.json"
+);
+
+impl From<UserOp> for entry_point_api::UserOperation {
```

Comment:
This is dup to the pr #24  . Just want to put it here to compile well. Should be removed later.